substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/IncompatibleSystemPropertyException.java

```diff
@@ -0,0 +1,7 @@
+package com.oracle.svm.core.jdk;
```

Comment:
License header missing on this file

Reply:
You could also just use the `UnsupportedFeatureError` class that we already have.